CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [4.10.0]
+
+### Added
```

Comment:
@vladanost Can you please add some more details to this, such as:

1. What support was added for the 2021 theme?
2. What does global customer login allow the user to do, high level. Something like "Added support for global customer login, with a default of on for all new installs and channels and the ability to turn on for existing channels."
3. More detail of when the checkout complete page displays / how it works

I'm trying to build a better developer enablement flow and having changelogs with clear details on what changed / was added is really helpful for that. Thanks!